packages/core/src/generators/qwik/src-generator.ts

```diff
@@ -453,9 +453,11 @@ export function isStatement(code: string) {
   }
   code = code.replace(STRING_LITERAL, 'STRING_LITERAL');
   const identifiers = code.split(EXPRESSION_SEPARATORS);
-  return (
-    identifiers.filter((i) => i && !i.match(EXPRESSION_IDENTIFIER)).length !== 0
-  );
+  const filteredIdentifiers = identifiers.filter((i) => {
+    i = i.trim();
+    return i && !i.match(EXPRESSION_IDENTIFIER);
```

Comment:
Small detail: I try to stray away from mutation, specially mutating function arguments. It's fine in this simple case, but we do a lot of mutations to append & build complex values, which I find hard to track.

What do you think?

```suggestion
    const trimmedI = i.trim();
    return trimmedI && !trimmedI.match(EXPRESSION_IDENTIFIER);
```

I might enable https://eslint.org/docs/rules/no-param-reassign on this repo 😁

Reply:
yes, agree. Let me change that.

Reply:
fixed